rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -178,6 +178,10 @@ def protocol_parameter_handling(self):
             "call_services_in_new_thread", False
         ).value
 
+        RosbridgeWebSocket.default_call_service_timeout = self.declare_parameter(
+            "default_call_service_timeout", 5.0
```

Comment:
Should the default be `5.0` or indefinite wait?

Reply:
The plan is to make it 5 seconds from Jazzy onward and indefinite when backporting it to Humble